shared/teams/emojis/common.tsx

```diff
@@ -42,20 +43,21 @@ export class AliasInput extends React.PureComponent<AliasInputProps, {}> {
             error={!!this.props.error}
             disabled={this.props.disabled}
             textType={Styles.isMobile ? 'BodySemibold' : 'Body'}
-            // android has issues with setSelection that make including colons a bad experience
-            // see https://keybase.atlassian.net/browse/TRIAGE-2680
-            value={Styles.isAndroid ? this.props.alias : `:${this.props.alias}:`}
+            value={`:${this.props.alias}:`}
```

Comment:
did you try the `transformText` stuff i mentioned?

Reply:
Oh I think I started to before the weekend but I didn't get super far, lemme try that stuff again